src/main/kotlin/me/schlaubi/intellij_gradle_version_checker/copy/ConvertGroovyCopyPasteProcessor.kt

```diff
@@ -43,9 +43,9 @@ import org.jetbrains.kotlin.psi.KtFile
 import java.awt.datatransfer.DataFlavor
 import java.awt.datatransfer.Transferable
 
-// https://regex101.com/r/kZ258U/3
+// https://regex101.com/r/1OMz6a/1
 private val DECLARATION_REGEX =
-    """(\w+)\s+(?:group:\s+)?["'](.+?)(?::|(?:(?:['"]),\s*(?:artifact:\s*)?(?:['"])))(.+?)(?:(?::|(?:(?:['"]),\s*(?:version:\s*)?(?:['"])))(.*))?(?:['"])""".toRegex()
+    """(\w+)\s+(?:group:\s*)?["']([^"':]+)(?::|["'],\s*(?:artifact:\s*)?['"])?([^"':]+)(?::|['"],\s*(?:version:\s*)?['"])?([^"':]+)?(?:['"])""".toRegex()
```

Comment:
Wouldn't it make sense to turn `:|['"]` into `[:'"]`?

Reply:
Why would it? `:` or `["'],\s*(?:artifact:\s*)?['"]`.

Reply:
Try the link in the changed codebase, for testing.